spec/controllers/alchemy/admin/pages_controller_spec.rb

```diff
@@ -598,6 +599,21 @@ module Alchemy
             alchemy_get :edit, id: page.id
           end
         end
+
+        context 'if page is scoped' do
+          context 'to a single role' do
+            context 'user role matches' do
+              before do
+                expect_any_instance_of(Page).to receive(:editable_by?).at_least(:once) { true }
+              end
+
+              it 'renders the edit view' do
+                alchemy_get :edit, id: page.id
+                expect(response).to render_template(:edit)
+              end
+            end
+          end
+        end
```

Comment:
Oh no 🙊 now the spec for "not letting in authorized users" has to be back in. Sorry 😁